### PR TITLE
Led viewer merge - Disable PSRAM for default allocations unless its the Mesmerizer

### PR DESCRIPTION
**Pull request by davepl**
Disable PSRAM for default allocations unless its the Mesmerizer.  This also fixes a number of range bugs where the matrix was always assumed to be 8-bit max width.

* [X] I read the contribution guidelines in [CONTRIBUTING.md](http://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/main/CONTRIBUTING.md).
* [X] I understand the BlinkenPerBit metric, and maximized it in this PR.
* [X] I selected `main` as the target branch.
* [X] All code herein is subjected to the license terms in [COPYING.txt](http://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/main/COPYING.txt).

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -327,6 +327,8 @@ build_flags     = -DLEDSTRIP=1
                   ${psram_flags.build_flags}
                   ${unity_double_flags.build_flags}
                   ${dev_adafruit_feather.build_flags}
+upload_port = /dev/cu.usbmodem8411101
+monitor_port = /dev/cu.usbmodem8411101
```
Comment (rbergen) [inline platformio.ini]: Personally, I think we should probably not include setup-specific ports in the checked-in platformio.ini.

```suggestion
```

**Review by rbergen — COMMENTED** [review]
One minor thing in platformio.ini I think we should not merge, but beyond that it looks good to me!

**Comment by davepl** [comment]
When it comes to checking my own serial port config into the project for convenience, I am reminded of the line from History of the World, Part 1:

“It’s good to be the King!”

…but yes, we omit it :-)

- Dave

**Comment by rbergen** [comment]
Hahaha, I'm just going to let that one go. :)

**Pull request merged**